packages/cozy-stack-client/src/PermissionCollection.js

```diff
@@ -155,6 +178,18 @@ class PermissionCollection extends DocumentCollection {
   }
 }
 
+/**
+ * Is the document a collection document?
```

Comment:
Nit: Can the description be clearer ? Explaining the context and not the what ?

"Is the document a collection document? " 

is not much clearer than

` export const isCollection = document => { `

I think it is because the stack needs to have "collection" in the name of the
attributes in the permission set ? Maybe we could rename this to reflect this
fact ? `isCollection` -> `needsToHaveCollectionNameInPermissionSet` ?

Reply:
👍